### ISSUE TITLE
Wine still has XQuartz as a dependency, despite not using it by default.

### ISSUE THREAD

**Issue by 1byte2bytes** [Homebrew/homebrew-core#15309]
# Please always follow these steps:
- [X] Confirmed this is a problem with `brew install`ing one, specific formula (not cask) and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Homebrew/brew/issues/new. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/caskroom/homebrew-cask/new
- [X] Ran `brew update` and retried your prior step?
- [X] Ran `brew doctor`, fixed all issues and retried your prior step?
- [X] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the [output link?](https://gist.github.com/anonymous/1139799566d5d601d8898aa4c816c8a7)
- [X] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

**Please note we will close your issue without comment if you delete or do not fill out the issue checklist and provide ALL the requested information.**

To help us debug your issue please explain:
- What you were trying to do (and why)
Install `wine` to run Windows Steam games on my MacBook Air
- What happened (include command output)
```
wine: XQuartz is required to install this formula.X11Requirement unsatisfied!

You can install with Homebrew-Cask:
  brew cask install xquartz

You can download from:
  https://xquartz.macosforge.org
Error: An unsatisfied requirement failed this build.
```
*later*
```
By default Wine uses a native Mac driver. To switch to the X11 driver, use
regedit to set the "graphics" key under "HKCUSoftwareWineDrivers" to
"x11" (or use winetricks).

For best results with X11, install the latest version of XQuartz:
  https://www.xquartz.org/
```
- What you expected to happen
I expected to not have to install extra unused dependancies
- Step-by-step reproduction instructions (by running `brew install` commands)
```
brew install wine
brew cask install xquartz
brew install wine
```

**Comment by JCount** [comment]
X11 is not an unused dependency, it is in fact recommended by upstream to help fill in some of the gaps that are still present in the native Mac driver.

From upstream:
> In the past, Wine's native Mac driver still needed a good deal of work so running Wine on macOS required installing an X11 server too. This is no longer necessary, but there are still some features missing from the Mac driver (see the to-dos on the macOS page) so you still might want to run Wine with the X11 driver.

> Installing WineHQ packages
Official WineHQ packages of the development and staging branches are available for macOS 10.8 and higher. Please test these packages and report any bugs at http://bugs.winehq.org.
Prerequisites:
XQuartz >= 2.7.7
Gatekeeper must not be set to block unsigned packages.

See: https://wiki.winehq.org/MacOS/Building https://wiki.winehq.org/MacOS

**Comment by JCount** [comment]
If you do not want to have X11 as an option, you can build it yourself using:
```brew install wine --without-x11```
